library/dnet.c

```diff
@@ -319,7 +319,7 @@ static void dnet_queue_wait_threshold(struct dnet_net_state *st)
 	/* If send succeeded then we should increase queue size */
 	while ((atomic_read(&st->send_queue_size) > DNET_SEND_WATERMARK_HIGH) && !st->__need_exit) {
 		/* If high watermark is reached we should sleep */
-		dnet_log(st->n, DNET_LOG_DEBUG,
+		dnet_log(st->n, DNET_LOG_ERROR,
```

Comment:
Because this is an error in the first place, and also this may help debugging the case, when elliptics suddenly eats all available ram without sending any single reply to clients.